pkg/agent/agent.go

```diff
@@ -246,73 +246,67 @@ func getAmbDiagnosticsInfo(url *url.URL) (*diagnosticsTypes.Diagnostics, error)
 }
 
 func (a *Agent) handleAPIKeyConfigChange(ctx context.Context) {
-	secrets, err := a.configWatchers.secretWatcher.List(ctx)
-	if err != nil {
-		dlog.Warnf(ctx, "Unable to list secrets for cloud connect token: %v", err)
-	}
-	cMaps, err := a.configWatchers.mapsWatcher.List(ctx)
-	if err != nil {
-		dlog.Warnf(ctx, "Unable to list configmaps for cloud connect token: %v", err)
-	}
-	a.setAPIKeyConfigFrom(ctx, secrets, cMaps)
-}
+	secret := &kates.Secret{ObjectMeta: metav1.ObjectMeta{
+		Name:      a.AgentConfigResourceName,
```

Comment:
Is the `cloudConnectTokenDefaultSuffix` case covered ?

Reply:
We don't actually need it because the value is set in that environment variable

Reply:
So yes its covered 🙂

Reply:
And did you check the rbac for ambassador cloud ?

Reply:
I don't see any changes in the chart :D